app/lib/bk/compat/parsers/harness.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+require_relative '../translator'
+require_relative '../pipeline'
+require_relative '../pipeline/step'
+require_relative './harness/steps/run'
+require_relative './harness/steps/background'
```

Comment:
No `background.rb` in harness/steps dir - `spec` passes with this commented out/removed

Reply:
:facepalm: this was left from some tests while doing some debugging (because obviously it took me quite a while to get this working)